network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/partner.html

```diff
@@ -1,30 +1,30 @@
 {% load wagtailimages_tags %}
 
+
 {% image page.partner_background_image fill-1200x600 as partner_img %}
-<img src="{{ partner_img.url }}" alt="{{ partner_img.alt }}">
-<div class="py-5">
+<div class="bg-secondary partner-section" style='background-image: url("{{ partner_img.url }}");'>
   <div class="container">
-      <div class="row">
-        <div class="col-12 mb-5">
-          <h2>{{ page.partner_heading }}</h2>
-          {% if page.partner_intro_text %}
-            <p>
-              {{ page.partner_intro_text }}
-            </p>
-          {% endif %}
-          <a href="{{ page.partner_page.url }}">
-            {{ page.partner_page_text }}
-          </a>
-        </div>
+    <div class="row">
+      <div class="col-12">
+        <h2 class="text-white">{{ page.partner_heading }}</h2>
+        {% if page.partner_intro_text %}
+          <p class="text-white partner-section-text">
+            {{ page.partner_intro_text }}
+          </p>
+        {% endif %}
+        <a href="{{ page.partner_page.url }}" class="text-white partner-section-button">
```

Comment:
```suggestion
        <a href="{{ page.partner_page.url }}" class="cta-link text-white">
```

and remove CSS rules for `partner-section-button`.

Note that `cta-link` will append arrow to the link text so we can remove the ` →` from line 16 as well.